mobile-academy/src/main/resources/sms-content.properties

```diff
@@ -0,0 +1,5 @@
+# localized content for completion sms (by language code)
+# TODO: update the language code suffix wherever necessary, assuming that 50 is hindi (national default)
+# TO add more languages, add as sms.content.61 or sms.content.72 where 61 and 72 would be the language codes
+# used in the deployment. (tamizh & punjabi for example)
+sms.content.50 = "Badhai ho didi, apne Mobile Academy course pura kar liya hai. Certificate paane ke liye is SMS ko apne subcenter meeting mein dikhayen. No "
```

Comment:
Ah ok. That space was specified by BBC as the format for the location number (No xxxxxxx) correlation.